liquid-rust-middle/src/core.rs

```diff
@@ -34,6 +35,7 @@ pub enum AdtDefKind {
 
 #[derive(Debug)]
 pub struct VariantDef {
+    pub def_id: DefId,
```

Comment:
Let's remove this because we are not sure we are populating it with the right `def_id`

```suggestion
```